parts/common/components.json

```diff
@@ -29,7 +29,7 @@
       "windowsVersions": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=windows/nanoserver",
-          "latestVersion": "1809",
+          "latestVersion": "ltsc2019",
```

Comment:
what is the benefit to upgrade to ltsc2019?

Reply:
Chatted offline, will revert to 1809 and add a comment why so the next developer doesn't upgrade it.